src/contracts/MultiCertifier.sol

```diff
@@ -19,10 +19,10 @@ contract Owned {
 contract Certifier {
 	event Confirmed(address indexed who);
 	event Revoked(address indexed who);
-	function certified(address) constant returns (bool);
-	function get(address, string) constant returns (bytes32) {}
```

Comment:
they're not pure in the interface; they're constant. switch `{}` for `;` instead and go back to `constant` rather than `pure`.